components/collective-page/CollectiveNotificationBar.js

```diff
@@ -47,9 +50,22 @@ const messages = defineMessages({
     id: 'collective.pending.description',
     defaultMessage: 'This collective is pending approval from the host ({host}).',
   },
+  'event.over.sendMoneyToParent.title': {
+    id: 'event.over.sendMoneyToParent.title',
+    defaultMessage: 'Event is over and still has a positive balance',
+  },
+  'event.over.sendMoneyToParent.description': {
+    id: 'event.over.sendMoneyToParent.description',
+    defaultMessage:
+      'If you still have expenses related to this event, please file them. Otherwise consider moving the money to your collective {collective}',
```

Comment:
There is a value named `{collective}` but seems it's not used, as illustrated in the last screenshot.